src/worker/start.rs

```diff
@@ -163,6 +165,17 @@ async fn resend_stdio(
     Ok(())
 }
 
+fn check_path_dirs(file: &StdioDef) -> io::Result<()> {
+    match file {
+        StdioDef::File(path) => match path.parent() {
+            Some(path) => std::fs::create_dir_all(path)?,
+            None => (),
+        },
+        _ => (),
+    };
+    io::Result::Ok(())
+}
+
```

Comment:
I would suggest to rewrite it slightly to remove the useless else branches. And also rename it, the current name is not very descriptive.
```rust
fn create_directory_if_needed(file: &StdioDef) -> io::Result<()> {
    if let StdioDef::File(path) = file {
        if let Some(path) = path.parent() {
            std::fs::create_dir_all(path)?;
        }
    }
    Ok(())
}
```